README.md

```diff
@@ -1,24 +1,89 @@
+- introduce new transactions associated with a category.
+- see the money spent on each category.
 
-* System dependencies
+## Views
 
-* Configuration
+> | -   | Splash Page                        | -   | Categories                          | -   | Purchases                           | -   |
+> | --- | ---------------------------------- | --- | ----------------------------------- | --- | ----------------------------------- | --- |
+> | -   | ![screenshot](app-screenshot1.png) | -   | ![screenshot](app-screenshot3.png) | -   | ![screenshot](app-screenshot2.png) | -   |
 
-* Database creation
+## Live App link
+- Here is the [link](https://budgety-rails-app.herokuapp.com/) to the live app `https://budgety-rails-app.herokuapp.com/`
 
-* Database initialization
+## objectives
 
-* How to run the test suite
+- Use ruby gems as software packages system.
+- Install Ruby on Rails framework.
+- Understand Rails RESTful design and router.
+- Use controllers to handle requests and render empty views.
+- Use params from browser request in a safe way.
+- Use preprocessed html file with embedded Ruby code.
+- Use layouts and templates for shared content.
+- Use database migration files to maintain database schema.
+- Use validations for models.
+- Secure app from n+1 problems.
+- Understand what ORM is.
+- Write SQL queries with ActiveRecord.
+- Set up associations between models.
+- Build a webapp that requyoures the user to log in.
+- Use devise gem for authentication.
+- Limit access to webapp resources based on authorization rules.
+- Analyze in writing why you have made a coding choice using one structure over another.
 
-* Services (job queues, cache servers, search engines, etc.)
+## Built With
 
-* Deployment instructions
+- Ruby on Rails
+- Gems
+  - Rspec
+  - Capybara
+  - FactoryBot
+  - Selenium-webdriver
+  - Letter-opener
+  - cancancan
+  - Devise
 
-* ...
+## Getting Started
+
+- Make sure you have Ruby installed in your system.
+- I can install it [here](https://www.ruby-lang.org/en/documentation/installation/).
+
+Clone this repository by running `https://github.com/billionsjoel/budgety.git` in your command line.
+
+Navigate to the repository by running `cd budgety`.
+
+## Installation
+
+- Clone the repo and run `bundle install` in the command line to install the dependencies.
+- run `rails s`
+
+## Video link
+
+`link to video: https://drive.google.com/file/d/1XTPb5Io7yJUponNJRVjrRYonRrP2dvJt/view?usp=sharing`
+
+## Author
+
+👤 **Atugonza Joel Billions**
+
+- GitHub: [@billionsjoel](https://github.com/billionsjoel)
+- LinkedIn: [Billionsjoel](https://www.linkedin.com/in/billionsjoel/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/billionsjoel/budgety/issues).
+
+## Show your support
+
+Give a ⭐️ if I like this project!
+
+## Acknowledgments
+
+- Coding partner, Standup and morning session team
```

Comment:
- [ ] Kindly note that one of the requirements says that you should give credit to the author of this design.[Gregoire Vella on Behance](https://www.behance.net/gregoirevella) the [author of the original design](https://www.behance.net/gallery/19759151/Snapscan-iOs-design-and-branding?tracking_source=), as required by the [Creative Commons license of the design](https://creativecommons.org/licenses/by-nc/4.0/).